x-pack/plugins/security_solution/public/timelines/components/create_field_button/index.tsx

```diff
@@ -22,6 +22,7 @@ import { sourcererSelectors } from '../../../common/store';
 import { useDeepEqualSelector } from '../../../common/hooks/use_selector';
 import { DEFAULT_COLUMN_MIN_WIDTH } from '../timeline/body/constants';
 import { defaultColumnHeaderType } from '../timeline/body/column_headers/default_headers';
+import { DataView, DataViewField } from '../../../../../../../src/plugins/data/common';
```

Comment:
I'd move this to line 13 to be on par with: https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/timelines/components/create_field_button/index.tsx#L13

to avoid future conflicts with imports.